modules/wificountry.sh

```diff
@@ -4,6 +4,7 @@ function wificountry {
   checkroot
   checkargn $# 1
   country=$1
+  country=${country^^}
   country_codes=(US CA JP DE NL IT PT LU NO FI DK CH CZ ES GB KR CN FR HK SG TW BR
```

Comment:
The country codes are missing Somalia (SO), European Union (EU), and the United Kingdom (UK):
![image](https://user-images.githubusercontent.com/61807745/114339730-c0b50880-9b0a-11eb-9cfe-57d4bd6e8eb1.png)

Reply:
1) Out of scope of the issue #2167
2) Those codes aren't built-in wificountry.sh so it always produces an error.
3) Even if I added the codes into the script, it would still not work. Check: -> http://www.lingoes.net/en/translator/langcode.htm